github/git_commits_test.go

```diff
@@ -12,15 +12,17 @@ import (
 	"net/http"
 	"reflect"
 	"testing"
+	"strings"
 )
 
 func TestGitService_GetCommit(t *testing.T) {
 	client, mux, _, teardown := setup()
 	defer teardown()
 
+	acceptHeaders := []string{mediaTypeGitSigningPreview, mediaTypeGraphQLNodeIDPreview}
 	mux.HandleFunc("/repos/o/r/git/commits/s", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		testHeader(t, r, "Accept", mediaTypeGitSigningPreview)
+		testHeader(t, r, "Accept", strings.Join(acceptHeaders, ", "))
```

Comment:
This is minor and either way is fine, but I'll make the suggestion for you to consider.

Since `acceptHeaders` is only used here, you can declare it right here. No need to declare it that much earlier:

```Go
acceptHeaders := []string{mediaTypeGitSigningPreview, mediaTypeGraphQLNodeIDPreview}
testHeader(t, r, "Accept", strings.Join(acceptHeaders, ", "))
```

Reply:
@shurcooL I added it this way so that in future if there is a need for more **mediaType** values we can just add it to the list, what do you think? Else as per your say I will change it :smile:

Reply:
@shurcooL  It was done to maintain consistency with the rest of the files when we had multiple mediaType values. Let me know if I should change it :smile:

Reply:
Okay, keeping it as is is completely acceptable.